### DISCUSSION TITLE
Be able to sort albums by exif date

### DISCUSSION THREAD

**Discussion by marcusisinger** [immich-app/immich#1854] [Feature Request]
### The feature

I migrated my library into Immich via cli and would like to be able to sort the albums by the exif date. Creation date of the album itself is not very useful. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by PeterBasista** [comment]
I also can't understand why sorting by album add date and title only... neither seems useful to me. Sorting by the time the photo in is probably the most important. Anyway, also, if anyone is in the business of uploading photos first and then going from top to bottom and creating albums (or skipping around like I did) I've made a fairly simple SQL update for that. The date of the last photo in the album will also be the date the album was created.

```
update albums al
set "createdAt" = (
    select a."fileCreatedAt" from assets a
    left join albums_assets_assets aaa on a.id = aaa."assetsId"
    where al.id = aaa."albumsId" 
    order by a."fileCreatedAt" desc 
    limit 1
    );
```

Just log into the immich db, check the names of the tables to see if they match, and run it. When you close the app and reopen it, the albums will load in the "correct" order.
